label_studio_tools/core/utils/io.py

```diff
@@ -54,7 +54,7 @@ def get_local_path(url,
 
     # File reference created with --allow-serving-local-files option
     if is_local_file:
-        filename, dir_path = url.split('/data/')[1].split('?d=')
+        dir_path, filename = url.split('/data/')[1].split('?d=')
         dir_path = str(urllib.parse.unquote(dir_path))
```

Comment:
Could you please change unquote for filename?

Reply:
If you mean replace `dir_path` with `filename` as an argument for `urllib.parse.unquote`, then I just made a commit for that.